src/log_batch.rs

```diff
@@ -616,8 +654,8 @@ where
         Some(vec)
     }
 
-    pub fn entries_size(&self) -> usize {
-        self.entries_size
+    pub fn approximate_size(&self) -> usize {
+        self.items_size + 20
```

Comment:
What's the meaning of 20?

Reply:
The log batch layout is `{ 8 bytes len | item count | multiple items | 4 bytes checksum }`